BD-3067: Engagement by Device report EA - MERGE 5/9 (#7304)
braze-inc/braze-docs: _docs/_hidden/private_betas/email_insights.md

---
nav_title: Email Insights - Engagement by Device
article_title: Email Insights - Engagement by Device
permalink: "/engagement_by_device/"
hidden: true
description: "This article provides an overview of the Engagement by Device report for the Email Insights dashboard."
---

# Email Insights dashboard

> The Email Insights dashboard helps you track and better understand where and when your customers are interacting with your emails. These reports can provide rich and granular data for marketers on how to optimize your emails to drive greater engagement. To access the dashboard, go to **Analytics** > **Email Performance** > **Email Insights**.

Reviewer: ```suggestion
The Email Insights dashboard helps you track and better understand where and when your customers are interacting with your emails. These reports can provide you with rich and granular data on how to optimize your emails to drive greater engagement. To access the dashboard, go to **Analytics** > **Email Performance** > **Email Insights**.
```

### Engagement by Device

The **Engagement by Device** report provides a breakdown of what devices your users are using to engage with your email. This data tracks email engagement across mobile, desktop, tablet, and other device types. 

> The "Other" category includes any user string that cannot be identified as desktop, mobile, or tablet. For example, but is not limited to television, car, video game console, OTT, or similar. This may also include null or empty values.

Reviewer: ```suggestion
The "Other" category includes any user string that cannot be identified as desktop, mobile, or tablet. For example, television, car, video game console, OTT, and similar. This may also include null or empty values.
```
